source/administration/upgrade.rst

```diff
@@ -0,0 +1,232 @@
+
+  On Ubuntu 16.04 and RHEL 7.1: ``sudo systemctl start mattermost``
+
+12. If you have TLS set up on your Mattermost server, you must activate the CAP_NET_BIND_SERVICE capability to allow the new Mattermost binary to bind to low ports.
+
+  1. ``cd {install-path}``
+  2. ``sudo setcap cap_net_bind_service=+ep ./bin/platform``
+
+After the server is upgraded, users might need to refresh their browsers to experience any new features.
+
+Upgrading Team Edition to Enterprise Edition
+--------------------------------------------
+
+To upgrade from the Team Edition to the Enterprise Edition, follow the normal upgrade instructions above, but make sure that you download the Enterprise Edition in Step 3.
+
+After the Enterprise Edition is running, open the *System Console* and go to **OTHER > Edition and License > License Key** and upload your license key file.
+
+Version Archive
+---------------
+
+Mattermost Enterprise Edition - Supported Versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost Enterprise Edition v3.9.0 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v3-9-0>`_ - `Download <https://releases.mattermost.com/3.9.0/mattermost-3.9.0-linux-amd64.tar.gz>`_
+  - ``https://releases.mattermost.com/3.8.2/mattermost-3.8.2-linux-amd64.tar.gz``
+  - SHA-256 Checksum: ``b99c86a2667f636eaee26331aa61a71a51b2d3d412eaa83fdebf8b53cddc6aeb``
+Mattermost Enterprise Edition v3.8.2 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v3-8-2>`_ - `Download <https://releases.mattermost.com/3.8.2/mattermost-3.8.2-linux-amd64.tar.gz>`_
+  - ``https://releases.mattermost.com/3.8.2/mattermost-3.8.2-linux-amd64.tar.gz``
+  - SHA-256 Checksum: ``b99c86a2667f636eaee26331aa61a71a51b2d3d412eaa83fdebf8b53cddc6aeb``
+Mattermost Enterprise Edition v3.7.4 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v3-7-4>`_ - `Download <https://releases.mattermost.com/3.7.4/mattermost-3.7.4-linux-amd64.tar.gz>`_
+  - ``https://releases.mattermost.com/3.7.4/mattermost-3.7.4-linux-amd64.tar.gz``
+  - SHA-256 Checksum: ``adc84d61e14812ff5bfd788be7e49f2e8b8d7803909fa83ce7e6b0f27cee2394``
+
+Mattermost Enterprise Edition - Unsupported Versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost Enterprise Edition v3.6.6 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v3-6-6>`_ - `Download <https://releases.mattermost.com/3.6.6/mattermost-3.6.6-linux-amd64.tar.gz>`_
+  - ``https://releases.mattermost.com/3.6.6/mattermost-3.6.6-linux-amd64.tar.gz``
+  - SHA-256 Checksum: ``257400926c3a20212a210fdeec2b0ef60f6845ed4075598b374c64e224a08b9f``
+Mattermost Enterprise Edition v3.5.1 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v3-5-1>`_ - `Download <https://releases.mattermost.com/3.5.1/mattermost-3.5.1-linux-amd64.tar.gz>`_
```

Comment:
It's actually 3.5.3: https://pre-release.mattermost.com/core/pl/d4n3fotc6pyairamhtyqntue5e

Our changelog says it's 3.5.1 though, so probably fine to just leave it as-is given it's an older version from last year.